src/main/java/org/jenkinsci/plugins/gitclient/GitClient.java

```diff
@@ -157,6 +157,15 @@
      */
     void add(String filePattern) throws GitException, InterruptedException;
 
+    /**
+     * rm.
```

Comment:
Could you add a sentence to describe the `git rm` command that is being invoked ?